### DISCUSSION TITLE
Increase amavis processes

### DISCUSSION THREAD

**Discussion by akito2001** [deeztek/Hermes-Secure-Email-Gateway#67] [Support]
We have deployed hermes seg but it appears to not be keeping up with our volume of mail.  The postfix queue keeps stacking up and says "(conversation with 127.0.0.1[127.0.0.1] timed out while receiving the initial server greeting)".  We believe this is due to the handoff to amavis and are trying to up the process count.  The config file doesnt appear to be at /etc/amavisd.conf like the amavis guide details.  Where was this file moved to?  I see /etc/amavis/conf.d but didnt see an equivelant filename in there.

**Comment by akito2001** [comment]
https://www.linuxbabe.com/mail-server/postfix-amavis-spamassassin-clamav-ubuntu

nano /etc/amavis/conf.d/50-user

that answered my question.  took a bit of time to finally find it

**Reply by deeztek** [reply]
The process count shouldn't be an issue unless you have very low resources assigned to that machine in terms of CPU and memory. It's possible it's an Amavis issue but I would need more information to definitely say. Regardless, you shouldn't make any permanent changes to any of the config files manually because the system will overwrite those changes whenever you make changes on the UI. I suggest you perform a SMTP test like below to see if you can send a message to yourself:

```
telnet 127.0.0.1 25
helo me
mail from: someone@domain.tld
rcpt to: someoneelse@domain.tld
data
subject: test
this is a test message
.
```
(ensure the last (.) dot is at the end of the test and ensure you use legitimate e-mail address for the "mail from" and "rcpt to".

**Reply by deeztek** [reply]
Also, please issue the following commands in the CLI and paste the results:

```
systemctl status postfix
systemctl status amavis
systemctl status djigzo
```

Thanks

**Comment by deeztek** [comment]
I'm going to go ahead and close this. Let me know if you are still having the issue.
